Cargo.lock

```diff
@@ -3876,9 +3876,12 @@ name = "rustc_session"
 version = "0.0.0"
 dependencies = [
  "log",
+ "num_cpus",
  "rustc_data_structures",
  "rustc_errors",
+ "rustc_fs_util",
  "rustc_index",
+ "rustc_target",
```

Comment:
This is unfortunate as this means that libsyntax now (again) depends on librustc_target via rustc_session whereas it didn't before (cause I managed to remove that edge). Even if it's not achievable right now with this PR, I think we should try hard to make libsyntax as "purely data" as possible, so let's try hard not to use many things in librustc_session from libsyntax.